pkg/session/manager.go

```diff
@@ -179,3 +204,37 @@ func (mgr *Manager) BlocklistMiddleWare(skipperFunc func() (echomiddleware.Skipp
 		}
 	}, nil
 }
+
+// extractUsername returns
+// - the current username from the token
+// - a bool flag that indicates whereas it's a built-in auth user
+// - an error
+func extractUsername(token *jwt.Token) (string, bool, error) {
+	content, err := extractContent(token)
+	if err != nil {
+		return "", false, err
+	}
+	sub, ok := content.Payload["sub"].(string)
+	if !ok {
+		return "", false, errExtractSub
+	}
+	username := strings.TrimSuffix(sub, ":login")
+	return username, len(sub) > len(username), nil
```

Comment:
Can't we instead check if the issuer of the token is `everest` (the `SessionManagerClaimsIssuer` const). Won't that be more reliable?

Reply:
A good point, I agree it's more reliable than checking the length. Updated, thanks!